src/main/java/com/tekenable/config/prime/trial/cancer/glaxo/GlaxoCancerPrimer.java

```diff
@@ -106,7 +106,7 @@ protected void createTrial() {
             add(trialInfo);
         }});
 
-        final TrialSite trialSite1 = new TrialSite(trialInfo,
+        final TrialSite trialSite1 = new TrialSite(trialInfo, 1,
```

Comment:
Why is it a logically better place? Yes this is still live, please update. 
It makes merging changes to the class a lot less likely to cause conflicts